src/apps/dashboard/dashboard-notification/dashboard-notification.tsx

```diff
@@ -31,7 +31,7 @@ const DashboardNotification: FC<DashboardNotificationProps> = ({
       onClick={notificationClickEvent}
       className="mb-16"
     >
-      <div className="list-dashboard shadow-medium-hover arrow__hover--right-small">
+      <div className="list-dashboard shadow-medium-hover arrow arrow__hover--right-small">
```

Comment:
Where is the `arrow` class in design system? Cant find it😬

Reply:
I'm not sure whether it's there, but I believer we use the arrow class together whenever we use arrow__hover--right-small